Automattic/wp-calypso
# Jetpack Cloud: Add placeholder for Main Scanner

#### Changes proposed in this Pull Request

* Update the placeholders on the scan page. 
To match the next content. 

Before:
<img width="932" alt="Screen Shot 2020-04-22 at 5 02 31 PM" src="https://user-images.githubusercontent.com/115071/79998513-10925e00-84bb-11ea-95e0-ead80cec4cc6.png">

After:
<img width="864" alt="Screen Shot 2020-04-22 at 4 02 17 PM" src="https://user-images.githubusercontent.com/115071/79998224-b7c2c580-84ba-11ea-95c1-1eda7215a5ad.png">


#### Testing instructions
* Navigate to the scan page. 
* Notice the new placeholder.
